src/dev/flang/parser/Lexer.java

```diff
@@ -1024,12 +1021,22 @@ <V> V bracketTermWithNLs(Parens brackets, String rule, Callable<V> c, Callable<V
     var end   = brackets._right;
     var ol = line();
     match(true, start, rule);
+    var indentRef = tokenPos();
     V result = relaxLineAndSpaceLimit(!currentMatches(true, end) ? c : def);
     var nl = line();
     relaxLineAndSpaceLimit(() ->
                            {
-                             match(true, end, rule);
-                             return Void.TYPE; // is there a better unit type in Java?
+                            if (current(true) != end._token)
+                              {
+                                // if indentation decreases before closing bracket, discard everything until closing bracket
```

Comment:
@fridis 
Is this what we want here or should/can we assume current indentation as new indentation and try to continue parsing?

Reply:
If there are braces, I discussed with @simonvonhackewitz that it makes sense to respect them and not mix this with indentation changes. So if a closing brace is missing, should be legit for the parser to skip until the end of the file.